pkgs/os-specific/darwin/apple-sdk-11.0/default.nix

```diff
@@ -24,16 +15,45 @@ let
       pbzx $src | cpio -idm
     '';
 
+    passthru = {
+      inherit (args) version;
+    };
+  });
+
+  MacOSX-SDK = mkSusDerivation {
+    pname = "MacOSX-SDK";
+    version = "11.0.0";
+
+    # https://swscan.apple.com/content/catalogs/others/index-11-10.15-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog
+    src = fetchurl {
+      url = "http://swcdn.apple.com/content/downloads/46/21/001-89745-A_56FM390IW5/v1um2qppgfdnam2e9cdqcqu2r6k8aa3lis/CLTools_macOSNMOS_SDK.pkg";
+      sha256 = "0n425smj4q1vxbza8fzwnk323fyzbbq866q32w288c44hl5yhwsf";
+    };
+
     installPhase = ''
       cd Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk
 
       mkdir $out
       cp -r System usr $out/
     '';
+  };
 
-    passthru = {
-      inherit version;
+  CLTools_Executables = mkSusDerivation {
```

Comment:
We only need CLTools_Executables for the ARC support library the Swift compiler links in. Swift requires SDK 10.15 at least, so there's no benefit doing it in the 10.12 SDK.

Reply:
Do the command-line tools need to stay in sync with the SDK? I'm not sure they shouldn't be an independent expression.

Reply:
Not necessary, but I'm not sure where else it'd belong. It can be a separate expression in a file in `pkgs/development/compilers/swift/compiler/`, but that seems out of place. Especially if we later find another use for it.

I was hoping we could also update these together and sourced from the same sucatalog, to prevent drift we later don't understand.

On the other hand, if we're going to use this as a template for newer SDKs, we'll have cruft in there that's not necessary right now. CLTools is only used from the SDK version the Swift compiler happens to use.